apps/intake/tests/utils/Paperwork.ts

```diff
@@ -209,6 +223,163 @@ export class Paperwork {
       secondaryInsuranceData,
     };
   }
+  async fillPaperworkAllFieldsTelemed(
+    payment: string,
+    responsibleParty: string
```

Comment:
nit: if there are several identical types in a row, it's better to use naming through destructuring to avoid accidentally making a mistake in the order of argument calls in client code